src/data/linux-quiz.ts

```diff
@@ -769,6 +769,16 @@ const linuxQuiz = [
       "The crontab command can used to perform various actions on cron jobs.",
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-crontab-command",
   },
+  {
+    Question: "Command which gives information on any hard drives that is connected to the system and their partitions.",
+    Answer: "fdisk",
+    Distractor1: "df",
+    Distractor2: "split",
+    Distractor3: "ps",
+    Explanation:
+      "fdisk also known as format disk is a dialog-driven command in Linux used for creating and manipulating disk partition table. It is used for the view, create, delete, change, resize, copy and move partitions on a hard drive using the dialog-driven interface. ",
```

Comment:
Let's shorten this a bit. 

```suggestion
      "The fdisk(format disk) is a command used to view, modify, delete, and copy partitions on a hard drive.",
```